examples/chat/main.go

```diff
@@ -54,33 +54,6 @@ var (
 const (
 	msgTimeout = time.Second * 10
 	timeLayout = "03:04:05 PM"
-
```

Comment:
Yeah, was commented just in case I was missing something since was unused.
Mentioned [here](https://github.com/textileio/go-textile-threads/pull/160#discussion_r355057448)

Reply:
I missed that one. Yep, not used... I must have forgotten to delete when tinkering with the example.